lib/spectacles.rb

```diff
@@ -40,4 +40,13 @@ def trailer(stream)
   end
 end
 
+ActiveRecord::ConnectionAdapters::SchemaStatements.class_eval do
+  alias_method(:_spectacle_drop_table, :drop_table) if method_defined?(:drop_table)
+
+  # Add this so that views get cleaned up properly
+  def drop_table(name, options={})
+    execute "DROP TABLE #{name} CASCADE"
```

Comment:
Oh, and do you happen to know if `CASCADE` is supported across the database adapters Spectacles supports?